### PR TITLE
cruft: Add version 2.10.1

### PR DESCRIPTION
**Pull request by astelmachonak-nydig**
Add a manifest for https://github.com/cruft/cruft

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by astelmachonak-nydig** [comment]
\verify

**Comment by rashil2000** [comment]
This is a pip package, right?

I would strongly recommend against managing such apps (this and cookiecutter) with Scoop.

For packages like these, language-specific package managers are much more capable and suited for installation/management.

See relevant discussion https://github.com/ScoopInstaller/Scoop/issues/3081 https://github.com/ScoopInstaller/Scoop/issues/4075

**Comment by rashil2000** [comment]
Hmm personally I don't have an issue with them, I just felt it would open up a can of worms with problems related to dependencies and updates. However you mention that they won't cause any problems, so I'll take your word.

Regarding the manifest, please don't use aliases in the scripts. Always use full cmdlet names, to avoid any possible conflicts with user aliases.

**Comment by astelmachonak-nydig** [comment]
@rashil2000 Thanks! I am active user of both packages, so I will monitor closely for any issues with them and promptly address them, however this packages are quite mature and doesn't get frequent updates. Let me get rid of aliases in both PRs

**Comment by astelmachonak-nydig** [comment]
@rashil2000 done for both, please take a look once you have a second

File: bucket/cruft.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            "    Remove-Item \"$fname\"",
+            "    deactivate",
+            "    Pop-Location",
+            "}"
+        ]
+    },
+    "bin": "cruft\\Scripts\\cruft.exe",
+    "checkver": {
+        "github": "https://github.com/cruft/cruft"
+    },
```
Comment (rashil2000) [inline bucket/cruft.json:27]:
```suggestion
    "checkver": "github",
```

**Comment by astelmachonak-nydig** [comment]
Thanks for improvement suggestions @rashil2000! I am still pretty new to scoop, so learning about tricks and best practices.

**Comment by rashil2000** [comment]
Sure, no worries, we're here to help :)

**Pull request merged**